.github/workflows/ci-pytorch-tests.yml

```diff
@@ -17,7 +17,6 @@ on:
       - "requirements/lite/*"
       - "src/lightning_lite/**"
       - ".actions/**"
-      - "!requirements/*/docs.txt"
```

Comment:
the same no need to run this CI if you just edit docs

Reply:
found answer in https://github.com/Lightning-AI/lightning/pull/15633/files#r1019732071